modules/log6x-logging-compatibility-support-matrix.adoc

```diff
@@ -0,0 +1,24 @@
+[id="log6x-logging-compatibility-support-matrix_{context}"]
+= Compatibility and support matrix
+
+Some features in this release are currently in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]. These experimental features are not intended for production use.
+
+In the table, features are marked with the following statuses:
+
+[horizontal]
+TP:: Technology Preview
+GA:: General Availability
+
+.Compatibility and support matrix
+[options="header"]
+|===
+
+| {clo} Version 6+| Component Version | OpenShift Version | Support Status
+
+| Operator | `eventrouter` | `logfilemetricexplorer` | `loki` | `lokistack-gateway` | `opa-openshift` | `vector` | |
+
+|6.0 | 0.4 (GA or TP) | v1.1 (GA or TP) | v2.9.4 (GA or TP) | v0.1 (GA or TP) | v0.1 (GA or TP) | v0.37.x (GA or TP) | 4.16 | GA or TP
+
+|===
+
+For questions and feedback, you can send an email to the product team at ...... .
```

Comment:
Do we have any email id to mention here? Like we have this ID for Pipelines: [pipelines-interest@redhat.com](mailto:pipelines-interest@redhat.com).

Reply:
@periklis @kabirbhartiRH, Could you please look into this question^^^^?

Reply:
@shipsing I don't know us having such an email for product requests. We usually ask people to file JIRA issues. There is no e-mail channel for questions, feedback or any sort of requests to the best of my knowledge. I suggest to talk to Jaime.